spec/models/cash_transaction_category_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe CashTransactionCategory do
+  let(:assessment) { create :assessment, :with_gross_income_summary }
+  let(:gross_income_summary) { assessment.gross_income_summary }
+
+  describe 'validation' do
+    let(:category) { described_class.new(gross_income_summary: gross_income_summary, operation: operation, name: name) }
+
+    context 'no name' do
+      let(:operation) { nil }
+      let(:name) { nil }
+      it 'is invalid' do
+        expect(category).not_to be_valid
+        expect(category.errors[:name]).to eq ["can't be blank"]
+      end
+    end
+
+    context 'operation credit' do
+      let(:operation) { 'credit' }
+      context 'invalid name' do
+        let(:name) { 'housing_costs' }
```

Comment:
should these :name attributes in this test come from CFEConstants::VALID_INCOME_CATEGORIES or  CFEConstants::VALID_OUTGOING_CATEGORIES rather than be manually input?